org.hedgewars.Hedgewars.yaml

```diff
@@ -11,7 +11,8 @@ finish-args:
   - --share=ipc
   - --socket=wayland
   - --socket=pulseaudio
-  - --device=all
+  - --device=dri
```

Comment:
Should probably be changed back to `--device=all`, but I am not sure how this is supposed to work so I may be wrong.

Reply:
Also, is this functionality included in a stable (1.16.x) Flatpak release?